packages/main/src/Slider.hbs

```diff
@@ -0,0 +1,36 @@
+<div 
+	id="{{this._id}}"
+	?disabled= "{{disabled}}"
+	value = "{{value}}"
+	label = "{{label}}"
+	expanded= "{{expanded}}"
+	@click = {{_onelementclick}}
+	@keydown={{_onkeydown}}
+	class = "sapMWS{{isExpandedClass}}{{isDisabledClass}}"
```

Comment:
better use attributes for CSS selectors, do not construct classes based on attributes that are already on the host and can be accessed